### PR TITLE
Add server code

### PR DESCRIPTION
**Pull request by suzil**
Thought I added this already but looks like I didn't.

### CODE REVIEW

File: server/.gitignore [hunk 1]
```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
```
Comment (suzil) [inline server/.gitignore:1]: Standard Python .gitignore

File: server/main.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+    max_wavenumber_range: float
+    species: List[Species]
+    pressure: float
+    tgas: float
+    tvib: Optional[float] = None
+    trot: Optional[float] = None
+    path_length: float
+    simulate_slit: bool
+    mode: str
+
```
Comment (arunavabasu-03) [inline server/main.py:34]: @suzil, I think now it's more clear for me.

File: server/main.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+    else:
+        if payload.simulate_slit:
+            spectrum.apply_slit(1.5, "nm")
+
+        wunit = spectrum.get_waveunit()
+        iunit = "default"
+        x, y = spectrum.get(payload.mode, wunit=wunit, Iunit=iunit)
+
+        # Reduce payload size
+        if len(spectrum) * 8 * 2 > 3e6:
```
[the remark this answers is not in the document]
Comment (erwanp) [reply server/main.py:75]: I think it's good to keep a maximum size ;  to have a fast, responsive app. Maybe 50 Mb is a good limit; which means we could increase the size by 5 or 10  ?

**Review by erwanp — APPROVED** [review]

File: server/main.py (hunk 2 above)
Comment (arunavabasu-03) [reply server/main.py:34]: Hey @erwanp, can we create a download for specific things?

**Comment by suzil** [comment]
Deployed to the server.

**Pull request merged**